chipy_org/apps/meetings/models.py

```diff
@@ -178,8 +178,6 @@ class Topic(CommonModel):
 
     def __str__(self):
         out = self.title
-        if self.presentors.count():
```

Comment:
This was causing a recursive call when adding a new topic. We may want to rethink this logic. Removing for now to unblock the page.